walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/IssuanceRequestContent.kt

```diff
@@ -26,7 +27,7 @@ internal class VerifiedIdRequestContent(
     // Root of trust of the requester (eg. linked domains).
     internal val rootOfTrust: RootOfTrust,
 
-    internal val injectedIdToken: InjectedIdToken? = null
+    internal val verifiedIDStyle: VerifiedIDStyle
```

Comment:
We need to sync on ID vs Id. I have "I lowercase d" throughout the iOS codebase. Which one do you prefer?

Reply:
I thought this was branding, that's why went this. I actually prefer Id. I will make the change.

Reply:
I changed in the external interface PR.